colcon_ros_bundle/task/ament_python/bundle.py

```diff
@@ -36,15 +36,27 @@ def add_arguments(self, *, parser):  # noqa: D102
                     'because it is in the workspace'.format_map(locals()))
                 continue
 
-            if 'version_eq' in dependency.metadata:
-                pip = args.installers['pip3']
+            symbol_mapping = {
```

Comment:
Should this include `~=` and `===`, as part of [PEP 440](https://www.python.org/dev/peps/pep-0440/#version-specifiers)?